Web Development/Simple Libray Management/readme.md

```diff
@@ -0,0 +1,49 @@
+# Simple Library Management
+
+A simple library management project that is easy to use and helps the learner to understand the basic concept of JS.
+
+Visit Website: [Online Library](https://delightful-coast-00b51a700.1.azurestaticapps.net/)
+
+<p align="center">
+<img src="https://user-images.githubusercontent.com/52650290/167833386-ec9951a2-fe1f-4c9b-88c9-2b7fe1ef6009.png">
+<img src="https://user-images.githubusercontent.com/52650290/167833479-370a3485-2286-4eed-a7a8-57bb2c5411ed.png">
+</p>  
+
+## Functionalities :-
+Input book name, author and book type to add the book. If book name is missing it will shown you error. As I used LocalStorage if you refresh or close the tab your books will not disappear. You can clear book shelf just by clicking on **clear shelf**. You can also search book by book name, author and type.
+
+### Adding Book in Shelf:
+
+<p align="center">
+<img src="https://user-images.githubusercontent.com/52650290/167833690-b5ac19a3-b49b-424a-b258-bae2bcb9dcb9.png">
+<img src="https://user-images.githubusercontent.com/52650290/167833986-826f5ea1-f403-4cc8-9200-2718c15f208f.png">
+</p>
+
+### Adding Book Without Entering Book Name:
+
+<p align="center">
+<img src="https://user-images.githubusercontent.com/52650290/167834120-d1dae43f-ab4c-46da-9f8f-fb688eed67b2.png">
+</p>
+
+### Clearing Book Shelf:
+
+<p align="center">
+<img src="https://user-images.githubusercontent.com/52650290/167834210-e4dfd335-d2fd-438b-904a-b1e3cb1625a2.png">
+</p>
+
+## Tech Stack :-
+- HTML
+- Bootstrap 
+- JavaScript
+
+## Demonstration :- 
+https://user-images.githubusercontent.com/52650290/170779099-6e592dfa-1f96-47e3-812f-568b506e2cb9.mp4
+
+## How to run :-
+![Dependencies](https://user-images.githubusercontent.com/78725162/158399068-9c84a79b-17d7-47cc-b1ec-02ee536cfcd2.png)
+![Clone](https://user-images.githubusercontent.com/78725162/158399122-5fcc235c-b2af-4607-ac48-5e97b9481f4b.png)
+

```

Comment:
Hey, you don't need to add that exact one, that's for reference how you need to add any dependencies that you have installed and also what commands to use for the quick start of the project to run the system locally.
If you are having any issues then do ping any of the mentors.

Reply:
Oh ok. My project not required any dependencies.